src/main/java/org/folio/services/kafka/topic/KafkaTopic.java

```diff
@@ -1,11 +1,20 @@
 package org.folio.services.kafka.topic;
 
-import java.util.stream.Stream;
+public class KafkaTopic {
+  public static KafkaTopic instance(String tenantId, String environmentName) {
+    return new KafkaTopic(prefixWith(environmentName,
+      prefixWith(tenantId, "inventory.instance")));
+  }
+
+  public static KafkaTopic holdingsRecord(String tenantId, String environmentName) {
+    return new KafkaTopic(prefixWith(environmentName,
+      prefixWith(tenantId, "inventory.holdings-record")));
+  }
 
-public enum KafkaTopic {
-  INVENTORY_INSTANCE("inventory.instance"),
-  INVENTORY_ITEM("inventory.item"),
-  INVENTORY_HOLDINGS_RECORD("inventory.holdings-record");
+  public static KafkaTopic item(String tenantId, String environmentName) {
+    return new KafkaTopic(prefixWith(environmentName,
+      prefixWith(tenantId, "inventory.item")));
+  }
```

Comment:
Maybe inline string concat will be sufficient.

Reply:
I deliberately kept them separate to try to emphasise the separate aspects of the name qualification. 

I'll change this to a single string statement.